index.html

```diff
@@ -112,51 +122,426 @@ <h2 class="section__sub-title-1">
-                            真実の愛によって解き放たれる二人の姉妹の物語
+                            国内史上最も愛されたウォルト・ディズニー・アニメーション・スタジオ映画がついに舞台化！真実の愛によって解き放たれる二人の姉妹の物語
                         </p>
                     </div>
                     <div class="section__des-wrapper">
                         <p class="section__des-txt-2">
-                            2020年9月、東京・竹芝エリアに誕生する新劇場・
-                            JR東日本四季劇場［春］。このオープニング作品として、
-                            ディズニー最新ミュージカル『アナと雪の女王』
-                            を上演いたします。2013年公開の劇場版長編アニメーショ
-                            ンを基に創作されたこの作品は、2018年3月、ブロードウェ
-                            イ初演。全世界のアニメーション映画興行収入歴代1位の記録を持
-                            つ名作の舞台化は、開幕するやいなや世界中で話題をさらいまし
-                            た。
-                        </p>
-                        <p class="section__des-txt-2 section__hide">
-                            ミュージカルでは、アニメーション版で監督も務めた脚
-                            本のジェニファー・リーが、アニメーションのストーリーラ
-                            インを踏まえながら、キャラクター造形や心情描写、
-                            それぞれの関係性をより丁寧に描き、奥行きあるドラマへとさ
-                            らに深化させました。作詞・作曲を手掛けるのは、
-                            「Let It Go」でアカデミー歌曲賞を受賞したロバート・
-                            ロペスとクリステン・アンダーソン＝ロペス夫妻。
-                            「生まれてはじめて」「雪だるまつくろう」などのオリジナル楽曲
-                            に加え、10曲以上もの新曲が追加されています。
-                        </p>
-                        <p class="section__des-txt-2 section__hide">
-                            凍った世界を救う真実の愛とは──絆で結ばれた二人
-                            の姉妹が織りなす物語は、再び日本に愛と感動を届けて
-                            くれることでしょう。また2020年は、劇団四季とディズニ
-                            ーの提携25周年という節目の年。この記念すべき年に、
-                            待望の超大作ミュージカル『アナと雪の女王』が、
-                            新劇場のグランドオープンを飾ります。
+                            2020年9月、東京・竹芝エリアに誕生する新劇場・JR東日本四季劇場［春］。このオープニング作品として、ディズニー最新ミュージカル『アナと雪の女王』を上演いたします。2013年公開の劇場版長編アニメーションを基に創作されたこの作品は、2018年3月、ブロードウェイ初演。全世界のアニメーション映画興行収入歴代1位の記録を持つ名作の舞台化は、開幕するやいなや世界中で話題をさらいました。
                         </p>
+                        <div class="section__hide">
+                            <p class="section__des-txt-2">
+                                ミュージカルでは、アニメーション版で監督も務めた脚本のジェニファー・リーが、アニメーションのストーリーラインを踏まえながら、キャラクター造形や心情描写、それぞれの関係性をより丁寧に描き、奥行きあるドラマへとさらに深化させました。作詞・作曲を手掛けるのは、「Let It Go」でアカデミー歌曲賞を受賞したロバート・ロペスとクリステン・アンダーソン＝ロペス夫妻。「生まれてはじめて」「雪だるまつくろう」などのオリジナル楽曲に加え、10曲以上もの新曲が追加されています。
+                            </p>
+                            <p class="section__des-txt-2">
+                                凍った世界を救う真実の愛とは──絆で結ばれた二人の姉妹が織りなす物語は、再び日本に愛と感動を届けてくれることでしょう。また2020年は、劇団四季とディズニーの提携25周年という節目の年。この記念すべき年に、待望の超大作ミュージカル『アナと雪の女王』が、新劇場のグランドオープンを飾ります。
+                            </p>
+                        </div>
                     </div>
                     <div class="section__see-btn-wrapper">
-                        <button>See more</button>
+                        <button class="intro__see-btn"></button>
```

Comment:
@Kyle-Rosal 

Do not include the text and the icon in the whole background image.
Use the actual text and the icon from the PSD layers.